src/lib.rs

```diff
@@ -1,5 +1,5 @@
+use ::tantivy as tv;
```

Comment:
I believe this is a needless change, feel free to correct me though.

Reply:
Without that change I get this during the build: 
```
error[E0659]: `tantivy` is ambiguous=> ] 177/178: tantivy
  --> src/lib.rs:2:5
   |
2  | use tantivy as tv;
   |     ^^^^^^^ ambiguous name
   |
   = note: ambiguous because of multiple potential import sources
   = note: `tantivy` could refer to a crate passed with `--extern`
   = help: use `::tantivy` to refer to this crate unambiguously
note: `tantivy` could also refer to the module defined here
  --> src/lib.rs:69:1
   |
69 | #[pymodule]
   | ^^^^^^^^^^^
   = help: use `crate::tantivy` to refer to this module unambiguously
   = note: this error originates in the attribute macro `pymodule` (in Nightly builds, run with -Z macro-backtrace for more info)
```

Reply:
Ah, I see, it's getting confused with the function `tantivy` and the module, weird that PyO3 now errors on this when it didn't use to.

This is fine to keep then 👍